### PR TITLE
Update App Cert page

### PR DESCRIPTION
**Pull request by RoAnnC1**
Removed link to outdated Sandbox registration page. Added contact information.

### CODE REVIEW

File: src/manage-apps/app-certification.markdown [hunk 1]
```diff
@@ -9,14 +9,22 @@ Handling data with care and delivering a positive experience for our customers i
...
-2. As you develop your app, your Certification Project Manager will record the exact APIs (including the version and verbs) that you will use for your application. See the requirements to pass certification in these Overview docs (including Support requirements expected of the Partner):
+Start with an overview of the whole process: [API documentation]({{ site.baseurl }}/api-reference/). During the Business Development phase, you will be provided access to a Professional Edition Sandbox. Sandboxes are required for partners or customers looking to use SAP Concur APIs to read and write data from SAP Concur. It is important that you contact your assigned technical resource at SAP Concur to gain access to a sandbox that best fits your scenario. If you are unsure of your technical contact, consider these 3 scenarios:
 
-Enterprise apps: <https://prezi.com/p/lw0qqy51zcmd/> (this does not pertain to a Triplink app integration)
-eReceipt apps: <https://prezi.com/p/ur2i1m8thynz/> (this does not pertain to a Triplink app integration)
+* Develop an application for the SAP Concur App Center. This requires a signed Partner Agreement, [contact us](mail to:concur_bizdev@sap.com).
```
Comment (sarraloew) [inline src/manage-apps/app-certification.markdown]: Remove the space in "mailto"

**Review by sarraloew — CHANGES_REQUESTED** [review]

File: src/manage-apps/app-certification.markdown (hunk 1 above)
Comment (RoAnnC1) [reply src/manage-apps/app-certification.markdown]: Fixed

**Review by sarraloew — APPROVED** [review]

**Comment by sarraloew** [comment]
:shipit:

File: src/_includes/header.html [hunk 2]
```diff
@@ -55,6 +55,7 @@
               <li><a href="https://forum.developer.concur.com/">Forum</a></li>
               <li><a href="{{ site.baseurl }}/tools-support/release-notes/index.html">Release Notes</a></li>
               <li><a href="{{ site.baseurl }}/announcements/index.html">Announcements</a></li>
+              <li><a href="{{ site.baseurl }}/announcements/videos.html">Videos</a></li>
```
Comment (sarraloew) [inline src/_includes/header.html]: should link to tools-support instead of announcements

**Review by sarraloew — CHANGES_REQUESTED** [review]

File: src/_includes/header.html (hunk 2 above)
Comment (RoAnnC1) [reply src/_includes/header.html]: Fixed link

**Review by sarraloew — APPROVED** [review]

**Comment by sarraloew** [comment]
:shipit:

**Comment by charlieconcur** [comment]
:shipit: Noting this is one we probably want to look at in preview.developer.concur.com before merging to livesite to make sure the iframe videos are what we expect.

**Comment by RoAnnC1** [comment]
Agreed, about confirming. I copied the formatting used for the video on the App Certification page and used StackEdit to review, but who knows what it will look like.

**Pull request merged**